docs/assets/dcr_tmux_simnet_setup.sh

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/env bash
```

Comment:
This script isn't committed with the execute bit set. Is that on purpose?

Reply:
Noticed the same as I had to `chmod` to run it.  I'm assuming its intentional since its placed in docs/assets, and I wouldn't expect an executable script there, still if that's the case, maybe we could add to the instructions to either copy it to the root directory of the repo (and add it to `.gitignore`) or copy it someone else local outside of the repo directory.

Reply:
Right.  I didn't set it because it's in `docs/assets`.  I debated whether or not to put it there, as I mostly think it doesn't belong, but I didn't see a better place to put it.  I'm open to suggestion here.  If it's outside of the docs directly, I can also commit it with the execute bit set.

Reply:
I updated this to create a new `contrib` folder, put the script there, and commit it with the executable bit set.  I also updated the doco link accordingly.